VideoLocker/res/layout/activity_course_base.xml

```diff
@@ -101,65 +101,62 @@
 
             <LinearLayout
                 android:id="@+id/last_access_bar"
-                android:orientation="vertical"
                 style="@style/last_access_bar_style"
-                android:visibility="gone">
```

Comment:
Hmm, it does look like the UI always shows the last accessed section even if the user has never visited the course before. Maybe open an issue on JIRA, and let it be verified by QA.
> On 29-Sep-2015, at 11:39 am, Mi@N Kh@LiD <notifications@github.com> wrote:
> 
> In VideoLocker/res/layout/activity_course_base.xml <https://github.com/edx/edx-app-android/pull/374#discussion_r40638683>:
> 
> >                  style="@style/last_access_bar_style"
> > -                android:visibility="gone">
> The view's visibility is set to gone in the last_access_bar_style
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/edx/edx-app-android/pull/374/files#r40638683>.
>